WinImageBuilder.psm1

```diff
@@ -351,14 +351,25 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]
-        [string]$osArch
+        [string]$osArch,
+        [parameter(Mandatory=$false)]
+        [switch]$develBranch=$true
+
     )
     Write-Host "Downloading Cloudbase-Init..."

```

Comment:
I would also believe a better way to address the Cloudbase-Init installation is to write a function for this.